<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>鼠标位置</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>421</width>
     <height>89</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="middle">
    <item>
     <widget class="QTextEdit" name="x_textEdit">
      <property name="font">
       <font>
        <family>Adobe Arabic</family>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="y_textEdit">
      <property name="font">
       <font>
        <family>Adobe Arabic</family>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>47</y>
     <width>421</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="top">
    <item>
     <widget class="QLabel" name="x_label">
      <property name="font">
       <font>
        <family>Adobe Arabic</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>X轴</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="y_label">
      <property name="font">
       <font>
        <family>Adobe Arabic</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Y轴</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>421</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="bottom">
    <item>
     <widget class="QPushButton" name="start_pushButton">
      <property name="font">
       <font>
        <family>Adobe Arabic</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>开始（Q）</string>
      </property>
      <property name="shortcut">
       <string>Q</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="end_pushButton">
      <property name="font">
       <font>
        <family>Adobe Arabic</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>结束（W）</string>
      </property>
      <property name="shortcut">
       <string>W</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
